server/src/sharepoint/sharepoint.service.ts

```diff
@@ -75,12 +75,15 @@ export class SharepointService {
     try {
       const { access_token } = await this.generateSharePointAccessToken();
       const SHAREPOINT_CRM_SITE = this.config.get('SHAREPOINT_CRM_SITE');
+      let [, relativeUrl] = folderIdentifier.split(SHAREPOINT_CRM_SITE);
 
-      // Escape apostrophes by duplicating any apostrophes.
-      // See https://sharepoint.stackexchange.com/a/165224
-      const formattedFolderIdentifier = folderIdentifier.replace(/'/g, "''");
-      const url = encodeURI(`https://nyco365.sharepoint.com/sites/${SHAREPOINT_CRM_SITE}/_api/web/GetFolderByServerRelativeUrl('/sites/${SHAREPOINT_CRM_SITE}/${formattedFolderIdentifier}')/${path}`);
-
+      // package related urls are provided as true relative urls — but the relative url for artifacts is not relative but
+      // only provided as full path
```

Comment:
Did you mean "// package relative urls are provided as true relative urls — but the url for artifacts is not relative. It is provided as a full path" ?

Honestly still confused what's going on here, though